src/components/button/button.tsx

```diff
@@ -39,8 +39,8 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
         ref={ref}
         type="button"
         className={buttonClasses}
-        disabled={isDisabled}
-        aria-disabled={isDisabled}
+        disabled={isDisabled || isLoading || undefined}
```

Comment:
Does it make sense to have a generic const just above the render function?
const disableButton =isDisabled || isLoading || undefined